Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,49 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the `users` group.
```

Comment:
That is GID 100, right? I'd mention the numeric value in the man page too.

Reply:
Yeah I think so, I'll have to double check.